vision/airdrop_vision_pipeline.py

```diff
@@ -0,0 +1,152 @@
+"""Runs vision code to find judges during an airdrop"""
```

Comment:
Not passing `pylint`.

Duplicate code issue, which you can just add an ignore comment for
```
vision/deskew/__init__.py:1:0: R0801: Similar lines in 2 files
==vision.airdrop_vision_pipeline:[64:76]
==vision.flyover_vision_pipeline:[64:76]
        for image_path in image_parameters.keys():
            if image_path not in completed_images:
                # Save the image path as completed so it isn't processed again
                completed_images.append(image_path)

                # Load the image to process
                image: consts.Image = cv2.imread(image_path)

                # Get the camera parameters from the loaded parameter file
                camera_parameters: consts.CameraParameters = image_parameters[image_path]

                # Find potential judges in the image (duplicate-code)
```

Reply:
See [https://github.com/MissouriMRR/SUAS-2023/blob/develop/flight/test_files/golf_waypoint_test.py#L85](https://github.com/MissouriMRR/SUAS-2023/blob/develop/flight/test_files/golf_waypoint_test.py#L85) for an example